### DISCUSSION TITLE
wal2json extension

### DISCUSSION THREAD

**Discussion by bhvngt** [supabase/supabase#2327] [Questions] [answered]
Does [wal2json](https://github.com/eulerto/wal2json) comes pre-installed in supabase? 

[supabase/postgres](https://github.com/supabase/postgres) readme mention that as part of the features. But when I use it as part of supabase `docker installation` for `local development`, it does not look it is installed. When I run following command (documented as per the wal2json repo site), I get `could not access file "wal2json": no such file or directory" error.

`SELECT 'init' FROM pg_create_logical_replication_slot('test_slot', 'wal2json');`

**Comment by dragarcia** [comment, marked as answer]
Hi @bhvngt!

It is not at the moment, but it will be for our next, upcoming release, `13.3.0` ([PR](https://github.com/supabase/postgres/pull/44)). Apologies for the confusion!

**Reply by bhvngt** [reply]
Thanks @dragarcia.
